src/btn.js

```diff
@@ -0,0 +1,17 @@
+function createBtn() {
+  const createBtn = document.createElement('button')
+  createBtn.innerHTML = 'play the game'
```

Comment:
Is it html or it's just text?

Reply:
I think it might be good to pass the text as a function parameter so we can have multiple buttons with various texts